src/shared/utils/uidl-utils.ts

```diff
@@ -101,6 +101,10 @@ export const traverseNodes = (
   if (node.type === 'conditional') {
     traverseNodes(node.content.node, fn, node)
   }
+
+  if (node.type === 'slot' && node.content.fallback) {
```

Comment:
we should make a switch here as we never run two if statements in the same function call for traverseNode. We also don't capture errors for unsupported types, right?

Reply:
yes, will change both traverseNodes and traverseElements